decidim-proposals/app/forms/decidim/proposals/admin/proposal_answer_form.rb

```diff
@@ -18,18 +18,8 @@ class ProposalAnswerForm < Decidim::Form
         validates :internal_state, presence: true, inclusion: { in: :proposal_states }
         validates :answer, translatable_presence: true, if: ->(form) { form.state == "rejected" }
```

Comment:
```suggestion
```

Reply:
This is not working, so we should drop it. Mind that this logic of rejected/accepted hardcoded should be removed as now the proposals states can be changed by admins.